man/systemd.network.xml

```diff
@@ -2598,6 +2598,14 @@ Token=prefixstable:2002:da8:1::</programlisting></para>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>UseHopLimit=</varname></term>
+        <listitem>
+          <para>Takes a boolean. When true, the hop limit received in the Router Advertisement will be
+          used. See <varname>IPv6HopLimit=</varname>. Defaults to true.</para>
+        </listitem>
+      </varlistentry>
+
```

Comment:
From RFC4861, section 6.3.4. Processing Received Router Advertisements:

```
If the received Cur Hop Limit value is non-zero, the host SHOULD set
its CurHopLimit variable to the received value.
```

Supporting Hop Limit is listed in the RFC as a "SHOULD" requirement. As such, the user ought not be given the ability to configure this. There seems to be only downsides to giving a user the ability to configure networkd so that it is no longer compliant with the standard.

In general, configuration options ought only be made available for an RFC's "MAY" requirements.

Reply:
This is OK as it's defaults to true. We did not had this option to be set and likely may break no clue. Hence OK to have a knob. We too also have knobs for similar configurations.